ext/ffi/lib.rs

```diff
@@ -507,6 +560,177 @@ async fn op_ffi_call_nonblocking(
     .unwrap()
 }
 
+fn op_ffi_ptr_of<FP>(
+  state: &mut deno_core::OpState,
+  buf: ZeroCopyBuf,
+  _: (),
+) -> Result<U32x2, AnyError>
+where
+  FP: FfiPermissions + 'static,
+{
+  let permissions = state.borrow_mut::<FP>();
+  permissions.check(None)?;
+
+  Ok(U32x2::from(buf.as_ptr() as u64))
+}
+
+fn op_ffi_buf_copy_into<FP>(
+  state: &mut deno_core::OpState,
+  (src, mut dst, len): (U32x2, ZeroCopyBuf, usize),
+  _: (),
+) -> Result<(), AnyError>
+where
+  FP: FfiPermissions + 'static,
+{
+  let permissions = state.borrow_mut::<FP>();
+  permissions.check(None)?;
+
+  let src = u64::from(src) as *const u8;
+  unsafe { ptr::copy(src, dst.as_mut_ptr(), len) };
```

Comment:
Suggestion: return an error if `dst.len() < len`?

Reply:
Actually this won't ever be a problem as the one place where it is called is:
```js
core.opSync("op_ffi_buf_copy_into", [
  packU64(this.pointer.value + BigInt(offset)),
  destination,
  destination.byteLength,
]);
```
and here the length is always the length of the `dst` variable, but implemented it anyways for good measure.